packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/aws-custom-resource/integ.aws-custom-resource-athena.ts

```diff
@@ -134,9 +134,20 @@ const notebook = new AwsCustomResource(stack, 'AthenaNotebook', {
 });
 notebook.node.addDependency(workgroup);
 
-new IntegTest(app, 'CustomResourceAthena', {
+const integTest = new IntegTest(app, 'CustomResourceAthena', {
   testCases: [stack],
   diffAssets: true,
 });
 
-app.synth();
+// Assertion to check if TestWG workgroup is present in the list of workgroups
+integTest.assertions.awsApiCall('Athena', 'listWorkGroups')
+  .expect(ExpectedResult.objectLike({
+    WorkGroups: [
+      {
+        Name: 'TestWG',
```

Comment:
`TestWG` is defined in a `workgroupName` variable above. We should reuse the variable rather than hardcoding here. That way if the value of `workgroupName` changes we will pick it up in the assertion too.